packages/babel-traverse/src/path/conversion.js

```diff
@@ -461,8 +461,13 @@ function getThisBinding(thisEnvFn, inConstructor) {
         if (supers.has(child.node)) return;
         supers.add(child.node);
 
-        child.replaceWith(
-          t.assignmentExpression("=", t.identifier(thisBinding), child.node),
+        child.replaceWithMultiple(
```

Comment:
It takes an array.

Reply:
@nicolo-ribaudo oops! I just fixed that. I don't fully understand the expected output in some of the tests, so I didn't catch that the second argument was being ignored.